### ISSUE TITLE
Clarify In-flight live / realtime telemetry 

### ISSUE THREAD

**Issue by overlandla** [hrishiballal/flight-logging-protocol-development#12]
The readme says something about 

> in-flight live / realtime telemetry

What does that mean? If this should be addressed in this protocoll, then the name "flight-logging" seems to be misleading.

**Comment by hrishiballal** [comment]
@overlandla Agreed, the idea here is to use this format to do both post-flight / during flight logging and also use it for telemetry in a context like InterUSS public portal. So I agree that the "flight-logging" name is not great, any suggestions?
